instrumentation/github.com/jackc/pgx/splunkpgx/sql.go

```diff
@@ -0,0 +1,120 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package splunkpgx provides instrumentation for the github.com/jackc/pgx
+// package when using database/sql.
+//
+// To use this package, replace any blank identified imports of the
+// github.com/jackc/pgx package with an import of this package and
+// use the splunksql.Open function as a replacement for any sql.Open function
+// use. For example, if your code looks like this to start.
+//
+//     import (
+//     	"database/sql"
+//     	_ "github.com/jackc/pgx"
+//     )
+//     // ...
+//     db, err := sql.Open("pgx", "postgres://localhost:5432/dbname")
+//     // ...
+//
+// Update to this.
+//
+//     import (
+//     	_ "github.com/signalfx/splunk-otel-go/instrumentation/github.com/jackc/pgx/splunkpgx"
+//     	"github.com/signalfx/splunk-otel-go/instrumentation/database/sql/splunksql"
+//     )
+//     // ...
+//     db, err := splunksql.Open("pgx", "postgres://localhost:5432/dbname")
+//     // ...
+package splunkpgx
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/jackc/pgx"
+	// Make sure to import this so the instrumented driver is registered.
+	_ "github.com/jackc/pgx/v4"
```

Comment:
I think it should be

```suggestion
	_ "github.com/jackc/pgx/v4/stdlib"
```